FWCore/Framework/interface/GetterOfProducts.h

```diff
@@ -116,6 +120,35 @@ There are some variants for special cases
 
 namespace edm {
 
+  template <typename U>
+  struct BranchTypeForContainerType {
+    static const BranchType branchType = InEvent;
```

Comment:
```suggestion
    static constexpr BranchType branchType = InEvent;
```
would a little bit better.

Reply:
Done.